api/krusty/replacementtransformer_test.go

```diff
@@ -546,3 +547,103 @@ metadata:
   name: red-dc6gc5btkc
 `)
 }
+
+func TestIssue4761_path_not_in_target_with_create_true(t *testing.T) {
```

Comment:
Could we have these two tests as tests for the replacements filter in `api/filters/replacement/replacement_test.go` instead of in krusty? The error is thrown by the filter, so I think it makes more sense to have it as a unit test for the filter, keeping the test closer to the logic that it is testing. 

The tests here in `api/krusty/replacementtransformer_test` are intended to cover specifically integration test cases that we wouldn't be able to test in the filter unit tests, such as base/overlay structures, name references, interactions with other fields, etc.

Reply:
Yep, will move them!